tools/text_processing/split_file_to_collection/split_file_to_collection.xml

```diff
@@ -16,6 +16,16 @@
         </xml>
         <xml name="numnew_fname">
             <param name="numnew" type="integer" label="Number of new files" min="1" value="1"/>
```

Comment:
in principle it would make more sense for `numnew` to also go into the conditional with `chunking` (so they are alternatives) but would it break continuity with the old version too much to hide an option behind a conditional?

Reply:
As long as we bump the tool version I think this is fine.

Reply:
Do you want to do this still @simonbray?

Reply:
yes, just fixing this and one other thing